utils/rng-tools/Makefile

```diff
@@ -53,7 +53,7 @@ define Package/rng-tools/install
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_BIN) ./files/rngd.init $(1)/etc/init.d/rngd
 	$(INSTALL_DIR) $(1)/etc/uci_defaults
```

Comment:
You could correct the same typo from this line, too:
uci_defaults  --> uci-defaults